app/assets/stylesheets/article-show.scss

```diff
@@ -963,129 +960,167 @@ article{
     border-top: 27px solid $green;
   }
 }
+.showpage-signin-cta {
```

Comment:
All the code within `.show-page-signin-cta` is dead code that's no longer used. It probably went back in because of some weird git issue. It'd be great if you could remove it. :)

Reply:
I'll do it now.

Reply:
This has now been removed as requested.